src/protocols/secure_channel/PairingSession.h

```diff
@@ -93,10 +109,12 @@ class DLL_EXPORT PairingSession : public SessionDelegate
      * Allocate a secure session object from the passed session manager for the
      * pending session establishment operation.
      *
-     * @param sessionManager session manager from which to allocate a secure session object
+     * @param sessionManager    Session manager from which to allocate a secure session object
+     * @param peerNodeId        If known, the scoped nodeid of the peer that should be considered if necessary
```

Comment:
See previous comments on naming.  And needs to clearly document how one indicates "unknown".